pkg/plugin/plugin.go

```diff
@@ -14,6 +14,9 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// These directories are filter out of the list reported to velero
```

Comment:
Is it possible to elaborate this comment?

Reply:
I will add something like the following: Velero doesn't allow other non-velero directories in the root of the object store, so these directories need to be filtered out. Longhorn creates `lost+found` automatically